[mqtt.homie] HomieThingHandlerTests.handleCommandRefresh test fails

The `HomieThingHandlerTests.handleCommandRefresh` test has been failing for some days now:

```
[ERROR] Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.459 s <<< FAILURE! - in org.openhab.binding.mqtt.homie.internal.handler.HomieThingHandlerTests
[ERROR] handleCommandRefresh(org.openhab.binding.mqtt.homie.internal.handler.HomieThingHandlerTests)  Time elapsed: 0.013 s  <<< FAILURE!
org.mockito.exceptions.verification.WantedButNotInvoked: 

Wanted but not invoked:
callback.stateUpdated(
    <Homie thing handler tests$$ lambda$ 9 0/ 0x 0 0 0 0 0 0 0 8 0 0 2 5 6 0 4 0>,
    <Homie thing handler tests$$ lambda$ 9 1/ 0x 0 0 0 0 0 0 0 8 0 0 2 5 6 4 4 0>
);
-> at org.openhab.binding.mqtt.homie.internal.handler.HomieThingHandlerTests.handleCommandRefresh(HomieThingHandlerTests.java:235)
Actually, there were zero interactions with this mock.

	at org.openhab.binding.mqtt.homie.internal.handler.HomieThingHandlerTests.handleCommandRefresh(HomieThingHandlerTests.java:235)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   HomieThingHandlerTests.handleCommandRefresh:235 
Wanted but not invoked:
callback.stateUpdated(
    <Homie thing handler tests$$ lambda$ 9 0/ 0x 0 0 0 0 0 0 0 8 0 0 2 5 6 0 4 0>,
    <Homie thing handler tests$$ lambda$ 9 1/ 0x 0 0 0 0 0 0 0 8 0 0 2 5 6 4 4 0>
);
-> at org.openhab.binding.mqtt.homie.internal.handler.HomieThingHandlerTests.handleCommandRefresh(HomieThingHandlerTests.java:235)
Actually, there were zero interactions with this mock.
```

Do you know what might be causing this test to fail and do you have a fix for this @J-N-K, @davidgraeff? 

If there is no easy solution it might be better to ignore it for now so full builds succeed again.

--- a/bundles/org.openhab.binding.mqtt.homie/src/test/java/org/openhab/binding/mqtt/homie/internal/handler/HomieThingHandlerTests.java
+++ b/bundles/org.openhab.binding.mqtt.homie/src/test/java/org/openhab/binding/mqtt/homie/internal/handler/HomieThingHandlerTests.java
@@ -45,6 +45,7 @@ import org.eclipse.smarthome.core.types.Command;
 import org.eclipse.smarthome.core.types.RefreshType;
 import org.eclipse.smarthome.core.types.TypeParser;
 import org.eclipse.smarthome.io.transport.mqtt.MqttBrokerConnection;
+import org.eclipse.smarthome.test.java.JavaTest;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
@@ -213,7 +214,6 @@ public class HomieThingHandlerTests {
 
     @SuppressWarnings("null")
     @Test
-    @Ignore("https://github.com/openhab/openhab2-addons/issues/6408")
     public void handleCommandRefresh() {
         // Create mocked homie device tree with one node and one read-only property
         Node node = thingHandler.device.createNode("node", spy(new NodeAttributes()));
@@ -232,8 +232,10 @@ public class HomieThingHandlerTests {
         thingHandler.device.nodes.put(node.nodeID, node);
 
         ThingHandlerHelper.setConnection(thingHandler, connection);
-        thingHandler.handleCommand(property.channelUID, RefreshType.REFRESH);
+        // we need to set a channel value first, undefined values ignored on REFRESH
+        property.getChannelState().getCache().update(new StringType("testString"));
 
+        thingHandler.handleCommand(property.channelUID, RefreshType.REFRESH);
         verify(callback).stateUpdated(argThat(arg -> property.channelUID.equals(arg)),
                 argThat(arg -> property.getChannelState().getCache().getChannelState().equals(arg)));
     }

